eventmon/monitord/impl.go

```diff
@@ -100,18 +102,27 @@ func (m *Monitor) updateNotifierList(logger log.Logger) {
 		logger.Printf("Deleting old keymaster server: %s\n", addr)
 		m.closers[addr] <- struct{}{}
 		delete(m.closers, addr)
+		delete(m.keymasterStatus, addr)
```

Comment:
dont you need a lock for the keymasterstatus?

Reply:
Oops. Yeah. Fixed.